DesignDoc.adoc

```diff
@@ -0,0 +1,69 @@
+= Terminal Fighter Design Document
+
+== Introduction
+
+The settings is in space and you are a fighter pilot in a futuristic ship. The only way to control the ship is through a keyboard, your "terminal". There is no movement control, like a Touhou type game where the ship is always moving forward. You only have control of the weapon systems. The game will consist of different levels where you can choose your weapons before starting the level, different weapons will be unlocked along the way.
+
+I'm thinking unlocking new weapons is done by completing difficult levels. You get ingame currency from different levels and you can use the ingame currency to get upgrades for weapons, ex. easier words, or missle system could get more missles.
+
+The player will have a shield and hitpoints. The shield will be done like in FTL. So it will regenerate quickly, but cannot block all types of damage. Hitpoints will be slowly regenerated, I'm thinking regeneration won't be linear, so the lower the player's hp is the slower the player would regenerate hitpoints.
+
+== Visual Theme
+
+The theme of the game will be similar to http://old.casualcollective.com/#games/The_Space_Game[the space game] by Casual Collective. Although the gameplay will be very differnet. The player will only see a radar so the graphics will be very simple, mcuh of the view will be the player's "terminal". Think basic shapes on a radar. I was thinking circle is the main character, triangles are enemies, squares are projectiles.
+
+== Controls
+
+All controls will be on the keyboard, even in the start screen.
+
+In gameplay you will select your weapon via hotkeys 1-5. Once a weapon has been selected by pressing a hotkey, the interface will switch to targeting mode where you type the names of enemies to target them, every weapon will be unique, and enemies will also do various things.
+
+== Weapons
+
```

Comment:
I'll add that additions to the design doc, but I won't change it in this one because it's not a "problem". These pull requests are more for discussing problems rather than new ideas.